src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -411,20 +396,16 @@ export class ExposureNotificationService {
     return this.finalize();
   }
 
-  private async getKeysFileUrls() {
+  private async getKeysFileUrls(): Promise<any> {
     const currentStatus = this.exposureStatus.get();
```

Comment:
Can `currentStatus` be passed into this function? A tiny bit more functional programming that way.